src/Appwrite/Utopia/Response.php

```diff
@@ -445,6 +445,20 @@ public function output(Document $document, string $model): array
         return $this->payload;
     }
 
+    public function file($data, $contentType, $date, $state): void
```

Comment:
- This could potentially go to the core framework.
- Indentation is off.
- No need to couple a core method with Appwrite specific behavior `X-Appwrite-Cache` can be at controller level. No need for `state` param. 
- Not sure this method works with bigger files.